### DISCUSSION TITLE
Rega Script bricht nach foreach loop ab ohne erkennbaren Grund

### DISCUSSION THREAD

**Discussion by cb0b** [OpenCCU/OpenCCU#2554] [🇩🇪 Fragen&Antworten] [answered]
Hallo Community,
gibt es einen Weg Rega Scripte zu debuggen? 
Ich laufe auf einen Fehler nach einer foreach Schleife (Script ist das hier: https://github.com/ioBroker/ioBroker.hm-rega/blob/master/regascripts/devices.fn). Im log auf der Raspimatic steht nur local0.err ReGaHss: ERROR: ScriptRuntimeError gefolgt vom script. 
RaspiMatic Version: 3.73.9 2023.11.30

**Comment by cb0b** [comment, marked as answer]
Problem gefunden: https://homematic-forum.de/forum/viewtopic.php?f=65&t=66673#p654381. 
Is halt extrem unschön, dass es da keinerlei Fehlermeldung gibt.

**Comment by Baxxy13** [comment]
Also hattes du ne Leiche in der ReGa? 
Was war es denn?
